app/schemas/models/product.schema.coffee

```diff
@@ -18,6 +18,7 @@ module.exports = ProductSchema = {
         properties: {
           code: { type: 'string' }
           amount: { type: 'integer', description: 'Adjusted cost in cents' }
+          duration: { type: 'string'}
```

Comment:
What is the format, effect of the duration?

Reply:
Will add a comment.  It's a Stripe duration, and we only support `once` currently.